teams/release/charter.md

```diff
@@ -0,0 +1,97 @@
+# Release Team Charter
+
+# Mission
+
+The release team manages the releases of the Rust project. This primarily consists of the following activities:
+
+* Regression triage (shared with T-compiler)
+* Release notes drafting
+* Release blog posts
+* Releasing itself (i.e. buttons, timing, process)
+* Whether to issue a stable point release?
+* Beta backport management (the act itself, not approval)
+
+Periodically, the release team discusses and consults others on the following things:
+
+* Target policies, such as deprecation and stabilization for tier-1 targets
+* Toolchain distribution (e.g., rustup <> dist builds interaction)
+
+## How does this further the Rust Project goals?
+
+* Release management is a core responsibility within the Rust project
+* Rust's success is dependent upon it's ability to effectively and regularly distribute updates to the compiler to our users.
+* The release team makes sure this process goes smoothly
+
+## Responsibilities & Commitments
+
+### [Decision Maker](../../common/darci.md#decision-maker)
+
+TODO
+
+* Whether to issue a stable point release?
+
+### [Accountable for results](../../common/darci.md#accountable-for-results)
+
+TODO
+
+* Triage subgroup responsibilities
+
+### [Responsible for work](../../common/darci.md#responsible-for-work)
+
+TODO
+
+* Releasing itself (i.e. buttons, timing, process)
+* Release notes drafting
+* Release blog posts
+* Beta backport management
+
+### [Consulted for input](../../common/darci.md#consulted-for-input)
+
+TODO
+
+* Regression triage (shared with T-compiler)
+* Target policies, such as deprecation and stabilization for tier-1 targets
+* Toolchain distribution (e.g., rustup <> dist builds interaction)
+
+### [Informed of change](../../common/darci.md#informed-of-change)
+
+TODO
+
+## How does this group make decisions?
+
+TODO
+
+## What work is not specifically part of maintainership?
+
+These activities may be carried out by contributors or maintainers, but are not considered core activities that ‘count’ towards [maintainership](https://github.com/rust-lang/governance/blob/master/common/membership_types.md#maintainership)
+
+* Attending weekly meetings
+* Release notes drafting
+* Beta backport rollups
```

Comment:
I think release notes drafting & beta backport rollups probably fit into maintainership -- they're not readily approachable by contributors I think